src/Script/admin.js

```diff
@@ -196,4 +211,34 @@ function updateUserLevel(userIdSelected)
         $('#admin-modal').modal('hide');
         loadActiveUsers();
     })
+}
+
+function editUserPrompt(userIdSelected)
+{
+    $("#admin-modal-header").children().remove()
+    $('#admin-modal-footer').children().remove()
+
+    $("#admin-modal-title").html("Update User");
+
+    let adminSelectLabel = $("<label>").html("Please select what you would like to edit");
+    let adminEditDiv = $("<div>", {class : "form-group modal-content-1"});
+    let adminSelecter = $("<select>", { id : 'editUserSelecter'});
+    let adminSelectoption1 = $("<option>").val(1).html('Please select a option');
+    let adminSelectoption2 = $("<option>").val(2).html('Update User Level');
+    let adminSelectoption3 = $("<option>").val(3).html('Reset Password');
+
+    $("#admin-modal-header").append(adminSelectLabel);
+
+    $(adminEditDiv).append(adminSelecter);
+    $(adminSelecter).append(adminSelectoption1, adminSelectoption2, adminSelectoption3);
+
+    $("#admin-modal-body").html("").append(adminEditDiv);
+
+    $('#editUserSelecter').on('change',(event) => {
+
+        if (event.target.value == 1) $('#admin-modal-footer').children().remove();
+         else if (event.target.value == 2) $("#admin-modal-footer").html("").append($("<button>", {class : "btn btn-danger", type : "text", id : "updateUserBtn1", onclick : `updateUserLevelPrompt(${userIdSelected})`}).html("Update User"));
+            else if (event.target.value == 3) $("#admin-modal-footer").html("").append($("<button>", {class : "btn btn-danger", type : "text", id : "updateUserBtn3", onclick : `passwordResetPrompt(${userIdSelected})`}).html("Reset Password"));
+                else null;
```

Comment:
why is null here? use property if statements with curly brackets

Reply:
fixed